pkg/controller/csi_handler_test.go

```diff
@@ -888,12 +1049,12 @@ func TestCSIHandler(t *testing.T) {
 		{
 			name:           "detach invalid PV reference-> error",
 			initialObjects: []runtime.Object{pvWithFinalizer(), node()},
-			addedVA:        deleted(vaWithNoPVReference(va(true, fin, ann))),
+			addedVA:        deleted(vaWithNoPVReferenceNorInlineVolumeSpec(va(true, fin, ann))),
```

Comment:
I think we're missing a test where both `PV` and `InlineVolumeSpec` are set. Hate to make you write another test after already writing this massive amount, thanks for making sure this is all well tested!

Reply:
Good catch, we should have an unit test to cover this case. Otherwise I can't find anything wrong in this PR.